src/modules/paste-manager.coffee

```diff
@@ -5,10 +5,20 @@ dom      = Quill.require('dom')
 Delta    = Quill.require('delta')
 
 class PasteManager
-  constructor: (@quill, @options) ->
+  constructor: (@quill, options) ->
     @container = @quill.addContainer('ql-paste-manager')
     @container.setAttribute('contenteditable', true)
     dom(@quill.root).on('paste', _.bind(this._paste, this))
+    @options = _.defaults(options, { containerToDelta: this._containerToDelta })
```

Comment:
Can you structure this similar to other modules where there's a `DEFAULTS` static: https://github.com/quilljs/quill/blob/develop/src/modules/image-tooltip.coffee#L10. Can we also name the config `onConvert` (that's the best I could come up with... might be a better name still but I think `containerToDelta` is too long and context dependent).